java/BUILD

```diff
@@ -6,96 +6,23 @@ java_library(
 )
 
 java_test(
-    name = "DecodingTest",
```

Comment:
What's the motivation behind this change?

Reply:
By mistake, no motivation, needs to be changed

Reply:
Will Rename

Reply:
you will have all the test at a single place.
you need to add in BUILD=> srcs and IntegrationTestSuite.
No need to add explict function for each test.

Reply:
See the content of BUILD file, now it becomes so less and easy to change.

Reply:
See the content of BUILD

Reply:
Thanks for the context. It's actually better to have a single target per test class as opposed to a single test target with all the test classes. Bazel will manage sharding for the tests more efficiently. Could you please revert these changes?

In addition, this isn't really an integration test. A collection of unit tests is different from an integration test.

Reply:
Reverted